projects/canopy/src/lib/card/card.component.scss

```diff
@@ -29,4 +40,105 @@
       border-bottom: none;
     }
   }
+
+  &.lg-card--orientation {
```

Comment:
I think this could be moved in an `orientation` directive so that it can be used by other components/scenarios too.

Reply:
Done, created a Orientation Directive. :)